lib/xapi-stdext-unix/unixext.ml

```diff
@@ -34,6 +34,27 @@ let mkdir_rec dir perm =
     mkdir_safe dir perm in
   p_mkdir dir
 
+(** removes a file or recursively removes files/directories below a directory without following
```

Comment:
We could add that providing a non-existent path will *not* lead to an error. So the behavior is similar to `rm -rf arg` in the shell.